core/embed/models/model_T2T1_layout.c

```diff
@@ -0,0 +1,81 @@
+#include "flash.h"
+#include "model.h"
+
+const flash_area_t STORAGE_AREAS[STORAGE_AREAS_COUNT] = {
+    {
+        .num_subareas = 1,
+        .subarea[0] =
+            {
+                .first_sector = 4,
+                .num_sectors = 1,
+            },
+    },
+    {
+        .num_subareas = 1,
+        .subarea[0] =
+            {
+                .first_sector = 16,
+                .num_sectors = 1,
+            },
+    },
+};
+
+const flash_area_t BOARDLOADER_AREA = {
+    .num_subareas = 1,
+    .subarea[0] =
+        {
+            .first_sector = 0,
+            .num_sectors = 3,
+        },
+};
+
+const flash_area_t BOOTLOADER_AREA = {
+    .num_subareas = 1,
+    .subarea[0] =
+        {
+            .first_sector = 5,
+            .num_sectors = 1,
+        },
+};
+
+const flash_area_t FIRMWARE_AREA = {
+    .num_subareas = 2,
+    .subarea[0] =
+        {
+            .first_sector = 6,
+            .num_sectors = 6,
+        },
+    .subarea[1] =
+        {
+            .first_sector = 17,
+            .num_sectors = 7,
+        },
+};
+
+const flash_area_t WIPE_AREA = {
+    .num_subareas = 3,
+    .subarea[0] =
+        {
+            .first_sector = 4,
+            .num_sectors = 1,
+        },
+    .subarea[1] =
+        {
+            .first_sector = 6,
+            .num_sectors = 9,
+        },
```

Comment:
Why is sector 15 skipped from the wipe area?

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/def5983ae4355ad4cbc4bb7dd0d4065baa4b356e

Reply:
So there is no reason to split it into two subareas, is there?

Reply:
actually, the reason for skipping sector 15 was MPU settings of bootloader, which makes the sector 15 not accessible. reverted the change and added a comment about this here: https://github.com/trezor/trezor-firmware/commit/e626bc52baf9179ee9bdc3be5716825c0ed8ee90

why is the MPU set this way, i dont know.

Reply:
Alright, now I have a different question. Shouldn't sector 15 be also skipped in `ALL_WIPE_AREA` here in this file a few lines below and also in `core/embed/trezorhal/unix/flash.c`?

(On a side note, I dislike the fact that the layout is defined twice in both these files. But this topic is already under discussion in another thread.)

Reply:
Now I realized it doesn't need to be skipped in `ALL_WIPE_AREA`, because `ALL_WIPE_AREA` is used by the boardloader only.